data/json/recipes/other/power_supplies.json

```diff
@@ -613,6 +633,26 @@
       [ [ "steel_chunk", 3 ], [ "scrap", 6 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "light_dry_cell",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
+    "skill_used": "chemistry",
+    "skills_required": [ "electronics", 2 ],
+    "difficulty": 2,
+    "time": "1 h",
+	"autolearn": true,
```

Comment:
Needs formatting.

Reply:
Please elaborate, the formating seems right to me... I do not understand.

Reply:
resolved